recipe/meta.yaml

```diff
@@ -1,46 +1,48 @@
-{% set version = "16.7.5" %}
+{% set version = "20.0.20" %}
 
 package:
   name: virtualenv
   version: {{ version }}
 
 source:
-  url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
-  sha256: b14f2548f9d9b3db3309756069d2df9cbb8eb1cbd5f968437fc4580a6dbf19b6
-  patches:
-    - 0001-Mach-O-Copy-any-dynamic-libraries-and-symlinks-to-th.patch
-    - 0002-ELF-Copy-any-dynamic-libraries-and-symlinks-to-the-c.patch
+  git_url: https://github.com/pypa/virtualenv.git
```

Comment:
```suggestion
  url: https://github.com/pypa/virtualenv/archive/{{ version }}.tar.gz
  sha256: f881be34c3178539aa7c78d3c26a1165b85f53ebbf9e1f82b1c84b6ea7742208
```

Reply:
This is a good suggestion, but we need the Git history because it uses `setuptools_scm` for versioning. 👍

Reply:
Indeed, the github tarballs will not have that information
but I think you can get the sdist package from ``pypi`` and the ``version.py`` file is already there

Reply:
That's the first thing I tried, but it tries to rebuild itself from source to install itself in a build environment (IIUC), and that did not work.